main.go

```diff
@@ -61,11 +65,16 @@ func main() {
 		if context.GlobalBool("debug") {
 			logrus.SetLevel(logrus.DebugLevel)
 		}
+		if path := context.GlobalString("debug-log"); path != "" {
```

Comment:
shouldn't this be under the `if "debug"` above to not log to file if no debug flag was passed?